docs/BEMAN_STANDARD.md

```diff
@@ -140,6 +140,22 @@ contain a one- or two-paragraph summary describing the library's purpose.
 **[CMAKE.PROJECT_NAME]** RECOMMENDATION: The CMake project name should be
 identical to the beman library name.
 
+**[CMAKE.PASSIVE_PROJECTS]** REQUIREMENT: CMake projects must not adjust
+user-specified compilation flags.
+
+User-provided compilation flags, whether specified via presets, command-line
+options, or toolchains, must not be modified by CMake projects. Therefore, CMake
+projects may not set variables that impact compilation flags such as
+`CMAKE_CXX_FLAGS` and `CMAKE_CXX_STANDARD`, as this would override user settings.
```

Comment:
Would this be sufficient to close #58 ?

Reply:
#58 suggests adding a FAQ entry about minimum C++ revision supported. I still think its a good idea to answer that question with "it depends, but this is most common..."

Reply:
👌 I could add this to my back log once this PR is merged.